python-backend/app/__init__.py

```diff
@@ -39,13 +41,17 @@ def create_app(test_config=None):
 
 
 def register_extensions(app):
+
     api.app = app
     api.init_app(app)
-
     cache.init_app(app)
     db.init_app(app)
     jwt.init_app(app)
-
+    if app.config.get('ENVIRONMENT_NAME') == 'prod':
+        sched.init_app(app)
+        if not app.debug or os.environ.get("WERKZEUG_RUN_MAIN") == 'true':
```

Comment:
Okay this takes away the concern I had partially. :)

Now for a single uwsgi context, this will only run on one main thread. But for our production instances, we have two separate pods serving the app.
Each pod has its own uwsgi context. So at the least, the scheduler will run twice for each thing. I dont have a good solution off the top of my head. But this will definitely be a concern we should look out for

Reply:
The way the jobs are set up it shouldn't be a problem if they run on both of the pods.